web/cobrands/fixmystreet-uk-councils/alloy.js

```diff
@@ -117,7 +117,7 @@ OpenLayers.Strategy.Alloy = OpenLayers.Class(OpenLayers.Strategy.FixMyStreet, {
 
 fixmystreet.assets.alloy_defaults = {
     http_options: {
-      base: "https://alloy-api-tile01.yotta.co.uk/api/render-layer/tile/${layerid}/28/${layerVersion}-/${z}/${x}/${y}",
+      base: "https://alloy-api-tile01.yotta.co.uk/api/render-layer/tile/${layerid}/${environment}/${layerVersion}-/${z}/${x}/${y}",
```

Comment:
The values that get interpolated in to `${layerVersion}` include the `-` suffix, so you end up with a double-dash in the URL e.g. `https://alloy-api-tile01.yotta.co.uk/api/render-layer/tile/308/28/308.8--/15/16303/10788`. Doesn't seem to matter but might be worth fixing whilst here?

Reply:
oooh, good spot, will remove that too.